eb_sqs/worker/worker.py

```diff
@@ -23,10 +23,15 @@ def __init__(self, queue_client, group_client):
         self.queue_client = queue_client
         self.group_client = group_client
 
-    def execute(self, msg):
-        # type: (unicode) -> Any
+    def execute(self, msg, receive_count=1):
+        # type: (unicode, int) -> Any
         try:
             worker_task = WorkerTask.deserialize(msg)
+
+            if receive_count > 1:
```

Comment:
Receive count should already be in the msg: `receive_count = int(msg.attributes[self._RECEIVE_COUNT_ATTRIBUTE])`

Reply:
In the message yes, but we only pass the message body into this method...
The param name is a little confusing